dev_notes/RELEASE_NOTES.md

```diff
@@ -0,0 +1,30 @@
+*(Please use the [release template](release-notes-template.md) as the template for this document)*
+<!-- Next release -->
+
+# MongoDB Kubernetes Operator 0.5.2
+## Kubernetes Operator
+* Changes
```

Comment:
We also added security context to the stateful set when `MANAGED_SECURITY_CONTEXT` is not set

Reply:
And we fixed an issue when the agent is ran with an non-root user